editor/editor_inspector.cpp

```diff
@@ -2424,10 +2456,14 @@ void EditorInspector::_notification(int p_what) {
 			if (refresh_countdown <= 0) {
 				for (Map<StringName, List<EditorProperty *>>::Element *F = editor_property_map.front(); F; F = F->next()) {
 					for (List<EditorProperty *>::Element *E = F->get().front(); E; E = E->next()) {
-						E->get()->update_property();
-						E->get()->update_reload_status();
+						if (!E->get()->is_cache_valid()) {
+							E->get()->update_property();
+							E->get()->update_reload_status();
+							E->get()->update_cache();
```

Comment:
If those three are always called together, I guess there could be a helper method to call the three of them?

Reply:
I think properties call them from within some times or its editors.